design-system/src/index.ts

```diff
@@ -5,6 +5,7 @@ export {
   themes,
 } from './design-tokens';
 export { ThemeProvider, useTheme } from './theme-provider';
+export type { ThemeName } from './theme-provider';
```

Comment:
Since we will need to modify `ui-kit` consumer components code based on the selected theme, I think it would be a good idea to export this type for better typing.